.gitignore

```diff
@@ -16,5 +16,9 @@ CMakeLists.txt.user
 # PyCharm generated files
 .idea/
 
-# VS Code system files
+# OS X hidden system files
+.DS_Store
+
+# VS Code configuration files
 .vscode/
+
```

Comment:
FWIW, this file shouldn't be part of this PR. It has already been modified as a result of another PR.